src/marshmallow_annotations/converter.py

```diff
@@ -49,7 +49,11 @@ class BaseConverter(AbstractConverter):
     def __init__(self, *, registry: TypeRegistry = registry) -> None:
         self.registry = registry
 
-    def convert(self, typehint: type, opts: ConfigOptions = None) -> fields.FieldABC:
+    def convert(
+        self,
+        typehint: type,
+        opts: ConfigOptions = None
+    ) -> marshmallow.fields.FieldABC:
```

Comment:
Not gonna block on this, but I tend to leave things how black formats them, in this case it's 

      def convert(
          self, typehint: type, opts: ConfigOptions = None
      ) -> marshmallow.fields.FieldABC: